doc/bc/changes-6.11.md

```diff
@@ -0,0 +1,20 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+
+- EZP-26885: Field Type external storage
+  - Field Type external storage gets directly injected with a proper external storage gateway.
```

Comment:
Ok, I wasn't sure how to indicate this. Maybe I just should keep `## Changes` section empty and keep `## Deprecations` only?